### PR TITLE
dev/entitypopulation

### PR DESCRIPTION
**Pull request by RobertRanson**
Populated entity classes.

### CODE REVIEW

**Review by AlanRobinson1337 — APPROVED** [review]
All entities set up with appropriate properties.
Constructors appropriate.
Getters and setters all correct

**Review by erinmaria98 — APPROVED** [review]
Looks good! Good use of BigDecimal.

File: src/main/java/org/somerville/swag/data/entity/Customer.java [hunk 1]
```diff
@@ -54,12 +86,12 @@ public void purchaseItems() {
...
 
-    public String getUsername() {
-        return username;
+    public String getEmail() {
+        return email;
     }
 
-    public void setUsername(String username) {
-        this.username = username;
+    public void setEmail(String email) {
```
Comment (hzl616) [inline src/main/java/org/somerville/swag/data/entity/Customer.java:93]: Great idea to make the username the email in this class.

**Review by sgoudham — APPROVED** [review]
Nice job using BigDecimal! However, I would recommend extracting the "£" out a constant variable for increased readability and to improve maintainability in the long run!

**Review by hzl616 — APPROVED** [review]
Nice job creating the general data for the Order, Product and Customer classes. The email address as username works well in Customer and meets MVP requirement

**Review by 2013884f — APPROVED** [review]
Nice Job

**Pull request merged**